lib/nailed/github.rb

```diff
@@ -1,14 +1,75 @@
+#
+# Nailed::Github
+#
 module Nailed
+  
+  #
+  # github helpers
+  #
+  def get_org_repos(github_client, org)
+    all_repos = github_client.org_repos(org)
+    all_repos.map(&:name)
+  end
+
+  def fill_db_after_migration(github_client)
```

Comment:
as I said in #29 i would keep `fill_db_after_migration` in `lib/nailed.rb` as it doesn't have a GitHub only context.

one could split it later on to `bugzilla` and `github` methods.